Documentation/async-disaster-recovery.md

```diff
@@ -0,0 +1,460 @@
+# Failover and Failback in RBD Async Disaster Recovery
```

Comment:
All the documentation files need a header such as this. The title is for the table of contents (and should be short), and the weight depends on where it should should up in the table of contents. What if we add it after the snapshots topic and before the volume cloning topic? In that case, we could set this to 3240, change snapshots to 3230, and leave volume cloning at 3250. 
```
---
title: Async DR Failover
weight: 3240
indent: true
---
```